surveys/static/js/location_create.js

```diff
@@ -0,0 +1,39 @@
+var location_area = $('*[id^="div_id_location-area-"]');
+var location_line = $('*[id^="div_id_location-line-"]');
+
+var location_line_date_measured = $('#id_location-line-date_measured');
+var location_area_date_measured = $('#id_location-area-date_measured');
+
+var typeDropdown = $("#id_location-geometry_type");
+
+
+$(document).ready(function() {
+  switchType(typeDropdown);
+});
+
+
+typeDropdown.change(function() {
+    switchType(this);
+});
+
+
+function switchType(typeDropdown) {
+  var selectedVal = typeDropdown.value
```

Comment:
Using a selectbox to control the conditional form is a smart way to start. One potential improvement we could make in the future would be to listen to the `L.Draw.Event.CREATED` event and switch the geometry type based on whether the user drew a line or a geometry. Do you think that would be useful, or can you imagine users wanting to create shapes that don't necessarily correspond to the PLDP `Line` or `Area` models?

I don't think we need make this improvement in this PR, but I want to get your take to see if I'm imagining a useful user workflow.

Reply:
I started going down this path and then realized, curiously, that while `geometry` is a required PLDP field `geometry_type` is not.

So theoretically someone could draw something and then not assign it a type. I'm not sure what the rationalization for this is, but since that's unclear I don't see a reason to limit users to picking one of those two types.

Reply:
Gotcha, thanks for that context!